src/backend/base/langflow/custom/custom_component/component.py

```diff
@@ -588,9 +588,9 @@ def _update_template(self, frontend_node: dict):
         return frontend_node
 
     def to_frontend_node(self):
-        #! This part here is clunky but we need it like this for
-        #! backwards compatibility. We can change how prompt component
-        #! works and then update this later
+        # ! This part here is clunky but we need it like this for
```

Comment:
Is the syntax `#!` used by something ? (conflicts with shebang btw)

Reply:
Not really. I use https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments and that makes it pop a bit more but not necessary. It is just to make sure people pay attention to certain comments more than others.

Reply:
Does it still highlight if you have a space before the `!` ?

Reply:
I think it should work and is recommended to have the space: https://github.com/aaron-bond/better-comments/issues/8#issuecomment-317007002

Reply:
Go for it. Thanks for the heads up!